dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassPathResult.java

```diff
@@ -111,5 +112,32 @@ public String formatDependencyPaths(Iterable<ClassPathEntry> entries) {
     }
     return builder.build();
   }
-  
+
+  /**
+   * Returns the class path entry for the artifact that matches {@code groupId} and {@code
+   * artifactId}. {@code Null} if no matching artifact is found.
+   */
+  ClassPathEntry findEntryById(String groupId, String artifactId) {
+    for (ClassPathEntry entry : getClassPath()) {
+      Artifact artifact = entry.getArtifact();
+      if (artifact.getGroupId().equals(groupId) && artifact.getArtifactId().equals(artifactId)) {
+        return entry;
+      }
+    }
+    return null;
+  }
+
+  /**
+   * Returns the class path entry that contains the class of {@code symbol}. {@code Null} if no
+   * matching entry is found.
+   */
+  ClassPathEntry findClassPathEntryForSymbol(Symbol symbol) throws IOException {
```

Comment:
findEntryBySymbol to parallel findEntryById

Reply:
Updated. Yes. ClassPathEntry is clear in the return type.